### ISSUE TITLE
Can't turn off bears

### ISSUE THREAD

**Issue by Iscariah** [Rikmuld/CampingMod-mcmod#40]
Hi!

For some reason, the option to turn off the bears isn't working. The bears annoyed me after a while and I tried to change it ingame to false via the mod configuration screen. The bears did still spawn, so I  did exit Minecraft to take a look at the config file and my change is there so I started the game again. When I saw a bear, I toggled to peaceful and the bear was gone. After toggling back to normal, I still found bears! So I did everything I could think of at this point...

I'm using version 2.1g for MC 1.7.10. I hope you could look into it. Thanks in advance.

**Comment by Iscariah** [comment]
Hi again!

I think that I figured it out already... When I posted my issue, I was tired and not thinking too clearly I suppose. It seems that the option works fine. The bears that have been already spawned stay. If I get rid of them by switching to peaceful, it will only remove the bears around me. Unloaded areas are unaffected by that and that's why I still find bears. Correct me if I'm wrong. ;) I did return a few times to places where I saw them regularly and they don't seem to return, which is what I wanted. I can live with it now. :+1: I'll also turn them back on when I've better armor and equipment because the venison is pretty good food.

**Comment by Rikmuld** [comment]
Ok great! I shall also add something that will make bears/foxes disappear, even if they are already spawned, when the respective mob is turned off.

**Comment by Iscariah** [comment]
Oh! But that sounds lovely! Thanks for taking the time to implement this. I really appreciate it, because I like to switch some things on and off from time to time. It depends on my mood and how far I am with a map. I recently began playing again on a new map and thought that it's a good idea to include your mod (which it is :D), but ofcourse I spawned in a forest surrounded by other types of forests. That's normally a great place to start, but not so much if you're armor-less with a big fat bear behind you. :P

Thanks again in advance for implementing that feature, I'm sure more people will benefit from this. At least it'll take some confusion away.
